app/src/main/java/org/simple/clinic/security/pin/PinEntryCardView.kt

```diff
@@ -138,11 +138,11 @@ class PinEntryCardView(context: Context, attrs: AttributeSet) : CardView(context
   }
 
   fun showIncorrectPinErrorOnSubsequentAttempts(remaining: Int) {
-    showError(resources.getString(R.string.pinentry_error_incorrect_pin_on_subsequent_attempts, remaining))
+    showError(resources.getString(R.string.pinentry_error_incorrect_pin_on_subsequent_attempts, "$remaining"))
   }
 
   fun showIncorrectAttemptsLimitReachedError(attemptsMade: Int) {
-    showError(resources.getString(R.string.pinentry_error_incorrect_attempts_limit_reached, attemptsMade))
+    showError(resources.getString(R.string.pinentry_error_incorrect_attempts_limit_reached, "$attemptsMade"))
```

Comment:
Both the string templates are unnecessary since there's only the variable. Can we use `.toString()` instead?

Reply:
See comment above.